apps/mobile/src/wallets/hooks/index.ts

```diff
@@ -151,31 +151,30 @@ export const useAddressHex = (wallet: YoroiWallet) => {
   const result = useQuery({
```

Comment:
if it was relying on suspense
```suggestion
  const result = useSuspenseQuery({
```